doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -23,13 +23,14 @@
 
 <a name="5.1"></a>
 ## 5.1 Introduction
-Securing Kubernetes requires several layers of security features to provide end to end security for cloud native applications. It is also important to adopt a layered packaging model which supports separation of concerns during image build. A fully integrated security testing phase should be baked into the CI/CD pipeline. Automated security policies should also be used to flag builds with issues.  Image registries must be monitored to automatically block or replace images with known vulnerabilites, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
+Securing Kubernetes requires several layers of security features to provide end to end security for cloud native applications. It is also important to adopt a layered packaging model which supports separation of concerns during image build. A fully integrated security testing phase should be baked into the CI/CD pipelines of both the vendors and operators. Automated security policies should also be used to flag builds with issues.  Image registries must be monitored to automatically block or replace images with known vulnerabilites, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
```

Comment:
Minor suggestion to avoid constraining to vendors and operators - i.e. 3rd parties, etc. working for either party. Slight wording change to avoid use of `must`/`should`, plus minor typo correction.

```suggestion
Securing Kubernetes requires several layers of security features to provide end to end security for cloud native applications. It is also important to adopt a layered packaging model which supports separation of concerns during image build. It is recommended that:
- Security testing is fully integrated into the CI/CD pipelines of all parties (e.g. vendors and operators).
- Automated security policies are used to flag builds with issues.
- Image registries are monitored to automatically block or replace images with known vulnerabilities, while also ensuring policies are used to gate what can be deployed and who can deploy from the registry.
```

Reply:
@tomkivlin Can we move, "It is also important to adopt a layered packaging model which supports separation of concerns during image build." under the recommendations as a separate bullet, "Adopt a layered packaging model which supports separation of concerns during image build."

Secondly, I think, "Securing Kubernetes ..." can be simply stated as "Kubernetes ..." as the securing part is covered with , "... to provide end to end security for cloud native applications."

Reply:
I'm not sure about that. Is it securing Kubernetes what requires several layers of security features or Kubernetes itself requires the several layers of security features. I think we are talking hare about the whole stack so I would keep the original wording.

Reply:
Layered packaging I moved to a bullet point.